public/js/main.js

```diff
@@ -71,9 +71,12 @@ const initCamera = async () => {
   }
 
   try {
+    const frameRate =
+      state.backend === 'cpu' ? { ideal: 10, max: 20 } : { ideal: 20, max: 30 };
     const stream = await navigator.mediaDevices.getUserMedia({
       video: {
         facingMode: 'user',
+        frameRate,
```

Comment:
Would you mind using a named argument for better readability?

Reply:
You mean, not using the [property shorthand](http://es6-features.org/#PropertyShorthand) on `frameRate`?

Reply:
oh I didn't realize property shorthand is a language feature - I thought we were mixing positional and named arguments which is usually not good practice. plz ignore!

Reply:
Oh, no worries.